docs/references/daemon.md

```diff
@@ -85,6 +85,7 @@ Version controlling of cluster manifests provides reproducibility and a historic
 | --k8s-secret-data-key                            | `identity`                         | data key holding the private SSH key within the k8s secret
 | **k8s configuration**
 | --k8s-allow-namespace                            |                                    | restrict all operations to the provided namespaces
+| --k8s-default-namespace                          | default                           | the namespace to use for resources where a namespace is not specified
```

Comment:
```suggestion
| --k8s-default-namespace                          |                                    | the namespace to use for resources where a namespace is not specified
```

Reply:
@moshloop I tried committing this change to save you hassle (and merge the PR), but I can't. Would you mind doing the honors?